### DISCUSSION TITLE
Compile to .app MacOS

### DISCUSSION THREAD

**Discussion by nn0ble** [neovide/neovide#885] [Q&A]
How do I turn the binary from ./target/release/neovide after compiling into a .app format? Do you use xcode and some sort of rust wrapper, or do you use [fruitbasket](https://github.com/mrmekon/fruitbasket) and if not would this be possible to implement without causing issues on other platforms when building

**Comment by lethanix** [comment]
You can use [Cargo bundle](https://github.com/burtonageo/cargo-bundle) and move the app to Applications folder:
```
cargo bundle --release
mv target/release/bundle/osx/Neovide.app /Applications/
```
